tests/OSPSuite.Starter/Forms/GridViewForm.cs

```diff
@@ -36,19 +43,47 @@ private void initializeBinding()
 
          var boundColumn = _gridViewBinder.Bind(x => x.Value)
             .AsReadOnly();
+
+
          boundColumn.XtraColumn.OptionsColumn.AllowFocus = true;
 
+
+         _gridViewBinder.Bind(x => x.ValueDescription);
+
+         _gridViewBinder.Bind(x => x.ValueOriginType)
+            .WithRepository(getRepository);
       }
 
-      private IEnumerable<Parameter> generateDummyContent()
+      private RepositoryItem getRepository(ParameterDTO arg)
       {
+         var repositoryItem = new UxRepositoryItemImageComboBox(gridView, _imageListRetriever);
+         repositoryItem.FillImageComboBoxRepositoryWith(ValueOriginTypes.AllValueOrigins, x => x.Icon);
+         return repositoryItem;
+      }
 
+      private IEnumerable<ParameterDTO> generateDummyContent()
+      {
          for (var i = 0; i < 10; i++)
          {
-            var parameter = new Parameter().WithName($"Prameter_{i}");
+            var parameter = new ParameterDTO().WithName($"Prameter_{i}");
             parameter.Formula = new ConstantFormula(i);
             yield return parameter;
          }
       }
    }
-}
+
+   class ParameterDTO : Parameter
+   {
+      public string ValueDescription
```

Comment:
Not sure if we will need those guys directly into the Parameter or not. This belongs more in a DTO but let's be ready to move that in Quantity if need be